ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -51,6 +51,12 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-aws-ovn-hybrid-custom-vxlan-port
+- as: vsphere-e2e-operator-with-custom-vxlan-port
```

Comment:
I suggest we call it `vsphere-e2e-operator`. Given it is vSphere it is well know that we need to use the custom VXLAN port.